pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "cgshop2025-visualization"
+version = "0.1.0"
+description = "A script for visualizing CGSHOP2025 data"
+authors = ["Damir Salakhov <damir-salakhov@mail.ru>"]
+license = "MIT"
```

Comment:
our license is GNU 3.0, but it doesn't matter much haha

Reply:
Changed